python/console/console_sci.py

```diff
@@ -428,6 +428,10 @@ def keyPressEvent(self, e):
             sys.stdout.fire_keyboard_interrupt = True
             return
 
+        #prevent accidental change in text size
+        if e.modifiers() & (Qt.ControlModifier) and e.key() in (Qt.Key_Plus, Qt.Key_Minus):
+            return
```

Comment:
Can you try to add a debug message in here (before the return) to see if the plus is caught as well?